solr/docker/templates/Dockerfile.official.header.template

```diff
@@ -90,5 +90,6 @@ RUN set -ex; \
   fi; \
   { command -v gpgconf; gpgconf --kill all || :; }; \
   rm -r "$GNUPGHOME"; \
-  tar -C /opt --extract --preserve-permissions --file "/opt/solr-$SOLR_VERSION.tgz";
+  tar -C /opt --extract --preserve-permissions --file "/opt/solr-$SOLR_VERSION.tgz"; \
+  rm "/opt/solr-$SOLR_VERSION.tgz"*;
```

Comment:
What's the asterisk for?

Reply:
It deletes the `.asc` file if one was downloaded